packages/wonder-blocks-core/components/clickable-behavior.js

```diff
@@ -35,8 +35,26 @@ type Props = {|
 
     /**
      * Passed in by withRouter HOC.
+     * @ignore
      */
     history?: any,
+
+    /**
+     * Trigger onClick callback and href navigation, if present, on enter key
+     * press. Default true. Only set to false if the component should definitely
+     * NOT trigger onClick on enter. An example of a component that shouldn't
+     * trigger on enter is Checkbox.
+     * @ignore
+     */
+    triggerOnEnter: boolean,
```

Comment:
Why are these marked as `@ignore`?  Someone might want to make a clickable `View` that behaves like a button but has an href.  I wonder if having a `role` property that allows people to set an aria role would be better.

Reply:
Okay, I decided to expose these.